news/8.feature.rst

```diff
@@ -0,0 +1 @@
+Exposed more `SessionOptions` from libbmq
```

Comment:
Will the information in the `news` folder be published somewhere?

Reply:
It makes it into the automatically built documentation using a tool called `towncrier` (and incidentally our CI yells at us if we don't add a news item for each PR!)